src/m365/graph/commands/changelog/changelog-list.ts

```diff
@@ -0,0 +1,196 @@
+import { Logger } from '../../../../cli';
+import GlobalOptions from '../../../../GlobalOptions';
+import { validation } from '../../../../utils';
+import AnonymousCommand from '../../../base/AnonymousCommand';
+import { Changelog, ChangelogItem } from '../../Changelog';
+import commands from '../../commands';
+import request from '../../../../request';
+import { DOMParser } from '@xmldom/xmldom';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  versions?: string;
+  changeType?: string;
+  services?: string;
+  startDate?: string;
+  endDate?: string;
+}
+
+class GraphChangelogListCommand extends AnonymousCommand {
+  private allowedVersions: string[] = ['beta', 'v1.0'];
+  private allowedChangeTypes: string[] = ['Addition', 'Change', 'Deletion', 'Deprecation'];
+  private allowedServices: string[] = [
+    'Applications', 'Calendar', 'Change notifications', 'Cloud communications', 
+    'Compliance', 'Cross-device experiences', 'Customer booking', 'Device and app management', 
+    'Education', 'Files', 'Financials', 'Groups', 
+    'Identity and access', 'Mail', 'Notes', 'Notifications', 
+    'People and workplace intelligence', 'Personal contacts', 'Reports', 'Search', 
+    'Security', 'Sites and lists', 'Tasks and plans', 'Teamwork', 
+    'To-do tasks', 'Users', 'Workbooks and charts'
+  ];
+
+  public get name(): string {
+    return commands.CHANGELOG_LIST;
+  }
+
+  public get description(): string {
+    return 'Gets an overview of specific API-level changes in Microsoft Graph v1.0 and beta';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['guid', 'category', 'title', 'description', 'pubDate'];
```

Comment:
That would indeed be more readable. Sad part is that we will lose the most valuable parameter in my opinion, the description. This one contains the most usefull information about the article/change.

Reply:
What's the use for the guid? Do we use it to let people get information about a specific change item? If yes, then we should keep it, but if there's no use for it, we should remove it to make room for `description`. Also, we should remove pubDate as it's pretty long and doesn't add much value in the overview.

Reply:
`guid` indeed has no essential usecase. So I'm all up for swaping it with description 😄 

For the `pubDate`, is it maybe an idea that we format it, only for text output, to a more readable string. For example: `Sat May 01 2021`

Reply:
What would a person consuming the change log use the date for? Let's keep in mind that it's the date of the announcement not the actual feature actually becoming available.

Reply:
I didn't know that the pubDate wasn't related towards the announcement of the feature. Then it has no use indeed.

Reply:
When we add `description` to the properties it will always output an unreadable output. 

![afbeelding](https://user-images.githubusercontent.com/38426621/187281802-e22f6a5e-4072-4917-8958-81f3de69aaa5.png)

What do you think @waldekmastykarz @Adam-it when we substring the `description` to 50 chars  when the output is in text mode?

![afbeelding](https://user-images.githubusercontent.com/38426621/187282074-00a7e3d4-50f9-4f61-b8f1-78d04f5a4d86.png)

Reply:
I'd say 50 charts + strip MD (for this we can use the same functionality that we use to render command's help in the terminal from our MD docs). To make it clear that the output was trimmed, let's add `...` at the end.

Reply:
Good idea, I was also thinking of adding the ellipsis overflow. 😄